Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: format-python format-notebook format-python-check format-notebooks-check test-notebook-parameters
+.PHONY: format-python format-notebook format-python-check format-notebooks-check test-notebook-parameters test-notebook-execution test-kfp-components test-notebooks test-all
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)